source/reference/minio-mc/mc-version.rst

```diff
@@ -78,7 +90,9 @@ Parameters
 
 .. mc-cmd:: ALIAS
 
-   *Required* The :ref:`alias <alias>` of a MinIO deployment and the full path
+   *Required*
```

Comment:
A little drive-by formatting to match the other items in this section.

`:required:` has a very different style, maybe that could be updated. (If it doesn't globally upset too many things.)

Reply:
Drive by fixes you note to do are always welcome!

I say it's better to go with `:required:` here and let's get the css fixed. Better than having to go back and do it later. We can live with a momentary problematic formatting until the CSS fixes go in.

@rushenn for your eyes.

Reply:
The existing parameters on this page don't have `:optional:` or `:required:` (only the one manually styled "required") so this would be a good time to deal with all of them together.

Reply:
Current code in repo uses the first, italic and following text below.